src/action/downloadCSV.ts

```diff
@@ -13,23 +14,25 @@ export enum EMIT_TYPES {
 }
 
 export const downloadCSV = async ({
-    newGamesOnly,
     url,
     emit,
     teamId,
+    name,
+    isSnoking,
 }: {
-    newGamesOnly: boolean;
     url: string;
     emit: EMIT_TYPES;
     teamId: string;
+    name: string;
+    isSnoking: boolean;
 }): Promise<void> => {
     let snoKingSeasonData: SnokingGame[] = [];
     const lazyErrorAlert = () =>
         alert("There was an issue processing the request. Try again later or email chris@chrisbenti.com");
 
     // Step 1: Get all of the games for the team from the SnoKing site.
     try {
-        snoKingSeasonData = await getSnokingSeasonData(url);
+        snoKingSeasonData = isSnoking ? await getSnokingSeasonData(url) : await getKhlSeasonData(url, name, teamId);
     } catch (e) {
         console.error({ error: e });
         //TODO: Better alert handling then just an ugly alert.
```

Comment:
Note for me: I should have my new error handling behavior for the teams URL also handle this sort of error

Reply:
https://github.com/snoking-to-benchapp-csv/snoking-to-benchapp-csv.github.io/issues/27